cli/internal/plugins/bzr/bzr_test.go

```diff
@@ -1 +1,54 @@
 package bzr
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/gobuffalo/buffalo-cli/plugins"
+	"github.com/stretchr/testify/require"
+)
+
+// testVersionRunner is a custom version runner used for testing purposes.
+type testVersionRunner struct {
```

Comment:
you don't need this type. you have `versionRunner` in `ifaces_test.go`. you can use that. :)